### PR TITLE
Mobile tooltip

### LINKED ISSUE #911: [Tooltip and Popover] Adjustments for mobile devices
https://share.goabstract.com/626e2e75-36ea-45d1-ba19-d76de4961160

Tooltips (Guidelines)

### PR DESCRIPTION
**Pull request by MateusStanki**
## Closes Issues

* Closes #911 

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

File: packages/input-phone/src/PhoneInput.js [hunk 1]
```diff
@@ -206,6 +206,7 @@ PhoneInput.defaultProps = {
   textSelectCountryCancel: "Cancel",
   textSelectCountrySearch: "Search",
   disabled: false,
+  fullScreenBreakpoint: "500px",
```
Comment (alcferreira) [inline packages/input-phone/src/PhoneInput.js]: Why 500px? 🤔

Comment (MateusStanki) [reply packages/input-phone/src/PhoneInput.js]: This value was already set, I just moved from a const to a prop

File: packages/tooltip/src/Tooltip.js [hunk 2]
```diff
@@ -35,11 +35,19 @@ const Container = styled.div`
     opacity: 0;
     transition: opacity 200ms ease-out;
   }
+
+  @media only screen and (max-width: 480px) {
```
Comment (vNakamura) [inline packages/tooltip/src/Tooltip.js]:
```suggestion
  @media only screen and (pointer: coarse) {
```
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer

Comment (MateusStanki) [reply packages/tooltip/src/Tooltip.js]: Nice, thanks!

File: packages/tooltip/src/Tooltip.story.js [hunk 3]
```diff
@@ -200,6 +201,55 @@ storiesOf("Tooltip/Tooltip", module)
...
+        <Tooltip
+          trigger={<Button type={buttonTypes.SECONDARY}>Mobile</Button>}
+          content="Use mobile breakpoint to see this popover on fullscreen."
+          positionX={POSITIONS.X.AUTO}
+          positionY={POSITIONS.Y.AUTO}
+          fullScreenBreakpoint="500px"
+        />
+        <Tooltip
+          trigger={<MdQuestion css="display:block;" />}
+          content="On mobile the minimun hit area is 40px height/width"
```
Comment (alcferreira) [inline packages/tooltip/src/Tooltip.story.js]:
```suggestion
          content="On mobile the minimum hit area is 40px height/width"
```

**Pull request merged**